frontend/src/reviews/ReviewDecisionDisplay.tsx

```diff
@@ -1,52 +1,81 @@
 import { EntityKind, ReviewResponse } from 'types/types'
 import { formatDateString } from 'utils/dateUtils'
+import { getRoleDisplay } from 'utils/roles'
 
 type ReviewDecisionDisplayProps = {
   response: ReviewResponse
+  modelId: string
 }
 
-export default function ReviewDecisionDisplay({ response }: ReviewDecisionDisplayProps) {
-  const isApproved = useMemo(() => response.decision === 'approve', [response.decision])
+export default function ReviewDecisionDisplay({ response, modelId }: ReviewDecisionDisplayProps) {
+  const { modelRoles, isModelRolesLoading, isModelRolesError } = useGetModelRoles(modelId)
 
+  const theme = useTheme()
+  const isApproved = useMemo(() => response.decision === 'approve', [response.decision])
   const username = response.user.split(':')[1]
 
+  if (isModelRolesError) {
+    return <MessageAlert message={isModelRolesError.info.message} severity='error' />
+  }
+
   return (
-    <Stack direction='row' spacing={2} alignItems='center'>
-      <UserAvatar entity={{ kind: EntityKind.USER, id: username }} size='chip' />{' '}
-      <Card
-        sx={{
-          width: '100%',
-          p: 1,
-        }}
-      >
-        <Stack direction='row' spacing={1} alignItems='center' sx={{ width: '100%' }} justifyContent='space-between'>
-          <Stack direction='row' spacing={1} alignItems='center'>
-            <Typography>
-              <UserDisplay dn={username} />
-              {` ${isApproved ? 'has approved' : 'has requested changes'}`}
-            </Typography>
-            {isApproved ? (
-              <Done color='success' fontSize='small' />
-            ) : (
-              <HourglassEmpty color='warning' fontSize='small' />
-            )}
+    <>
+      {isModelRolesLoading && <Loading />}
+      <Stack direction='row' spacing={2} alignItems='center'>
+        <UserAvatar entity={{ kind: EntityKind.USER, id: username }} size='chip' />{' '}
+        <Card
+          sx={{
+            width: '100%',
+            p: 1,
+          }}
+        >
+          <Stack
+            direction={{ xs: 'column', sm: 'row' }}
+            spacing={1}
+            alignItems='center'
+            sx={{ width: '100%' }}
+            justifyContent='space-between'
+          >
+            <Stack alignItems={{ xs: 'center', sm: 'flex-start' }} spacing={{ xs: 1, sm: 0 }}>
+              <Stack direction={{ xs: 'column', sm: 'row' }} spacing={1} alignItems='center'>
+                <Typography>
+                  <UserDisplay dn={username} />
+                  {` ${isApproved ? 'has approved' : 'has requested changes'}`}
+                </Typography>
+                {isApproved ? (
+                  <Done color='success' fontSize='small' />
+                ) : (
+                  <HourglassEmpty color='warning' fontSize='small' />
+                )}
+                {response.outdated && (
+                  <Typography sx={{ backgroundColor: theme.palette.warning.light, borderRadius: 1, px: 0.5 }}>
```

Comment:
Subjective, but I think the standard `theme.palette.warning.light` looks better. The colours we use elsewhere are vibrant and this looks much more dull in comparison. What do you think of using a border like this?

![image](https://github.com/gchq/Bailo/assets/110473008/5ed6184f-292c-4430-b901-6fb6ac2463d2)